### Data Management 

<!-- * General use case description(s)
* Specific tool examples 
* Alternate solutions/ why is it better with BrAPI
* future related use cases, areas to improve -->

#### PHIS 

The Hybrid Phenotyping Information System ([PHIS](https://nph.onlinelibrary.wiley.com/doi/10.1111/nph.15385)), based on the [OpenSILEX](https://github.com/OpenSILEX/) framework, assumes responsibility for the systematic collection and management of data from phenotyping and high-throughput phenotyping experiments on a day-to-day basis. PHIS has the ability to efficiently store, organize and manage a wide range of data sets, including images, spectra and growth curves. This functionality extends to data at multiple spatial and temporal scales, from leaf to canopy, from a variety of sources such as field and greenhouse environments.

A key feature of PHIS is the unambiguous identification of all objects and traits within an experiment, establishing consistent relationships between them through the application of ontologies and semantics. This approach is designed to adapt to variations in experimental conditions, whether in the field or in controlled environments. PHIS's ontology-driven architecture emerges as a robust tool for integrating and managing data derived from diverse experiments and platforms, facilitating the creation of meaningful relationships between objects and augmenting datasets with relevant knowledge and metadata.

Furthermore, PHIS adheres to the Minimal Information About a Plant Phenotyping Experiment ([MIAPPE](https://doi.org/10.1111/nph.16544)) and the Breeding API (BrAPI) standards. 

The system recommends specific naming conventions, fostering a standardized approach for users to declare their resources. Notably, PHIS is widely adopted by various experimental platforms of the national [PHENOME](https://www.phenome-emphasis.fr/) and European [EMPHASIS](https://emphasis.plant-phenotyping.eu/) infrastructure, serving as a hub for data management. 
Moreover, dedicated instances of PHIS have been established for the explicit purpose of resource sharing, encompassing projects, genetic resources, and variables, thereby fostering collaborative engagement and the dissemination of knowledge pertaining to studied concepts.

##### PHIS & BrAPI

PHIS offers a RESTful API designed to streamline interaction with data within a platform. Within this API, various services aligning with the Breeding API (BrAPI) standards have been implemented, encompassing the Core, Phenotyping, and Germplasm modules. Comprehensive documentation for these services is available on the PHIS Swagger interface. This integration with Swagger ensures that users can easily access, understand, and utilize the functionalities provided by the BrAPI-compliant web services, fostering transparency and facilitating effective engagement with the PHIS platform.

The design and development of PHIS have been meticulously tailored, incorporating the explicit constraint of aligning with BrAPI requirements. This intentional alignment ensures that PHIS adheres to the specified standards and protocols outlined by the Breeding API (BrAPI), thereby fostering seamless integration and compatibility with BrAPI-compliant systems and platforms. This prerequisite served as a substantial foundation for formalizing the data model, simultaneously facilitating compatibility with other standards, such as MIAPPE. Thus, by consciously incorporating BrAPI requirements into its structure, PHIS not only meets the phenotyping domain standards but also enhances its capacity for interoperability and effective collaboration within the broader context of plant breeding and related domains.

The fact that data within a PHIS instance can be queried through BrAPI services makes the indexing of PHIS in [FAIDARE](https://urgi.versailles.inra.fr/faidare/) very easy to implement.

Indeed, as PHIS offers BrAPI-compliant Web Services, this greatly simplifies the integration and data exchange with other European information systems that handle phenotyping data. The adherence to BrAPI standards ensures a common interface and compatibility, facilitating seamless communication and collaboration between PHIS and other systems within the European context. This interoperability not only streamlines data sharing but also promotes a more cohesive and effective approach to managing and utilizing phenotyping data across diverse platforms and research initiatives in the European scientific community.

#### DeltaBreed

<!-- Peter S: Stub paragraph to stimulate the writing process. Please edit, rewrite, or delete as needed. -->
DeltaBreed is a tool built almost entirely based on the BrAPI standard. It was born out of a recognized need to 1) unify open-source software, R packages, and pipeline scripts in the plant and animal breeding space with a single API (i.e., BrAPI) and 2) to provide users with a breeder-friendly and species-agnostic UI for transactional breeding data management in public breeding programs.  The need for the former unification stems from the fact that many public breeders are running very small teams on very thin budgets and simply cannot become experts in all new technologies that would benefit their breeding programs if integrated.  The latter was identified through gap analyses spanning diverse niche species, specifically specialty crops and livestock, that have breeding data management needs outside of current software applications focused on major crops and livestock breeding.

Specialty crop and livestock breeding programs are typically public programs that mainly rely on phenotypic selection.  While progress is made, it is a slow and imprecise method to make genetic gains.  Adopting new technology piecemeal also is slow and can lead to dissatisfaction if the benefit is not substantial.  Rather than attempt to train every public specialty breeder to be a genomicist, bioinformaticist, statistician, phenomicist, data analyst, and several other disciplines, DeltaBreed was designed to be the top layer UI that interacts via BrAPI in the backend to connect multiple modular software components (*Figure x*).  When fully developed, DeltaBreed will allow any breeder to access tools and pipelines that would otherwise be difficult to adopt, all via BrAPI.

Breeding Insight decided to build DeltaBreed to be species-agnostic due to missing management features in available open-source software like BreedBase, BMS, BIMS, and EBS.  These software packages were designed with major staple crops like corn, wheat, and rice in mind, and therefore have features tuned to the needs of these crops.  By sourcing use cases for DeltaBreed more widely across species, Breeding Insight pinpointed features required for technology adoption not currently addressed by existing software while simultaneously achieving an economy of scale in design.  Some missing tool examples include support for rootstock, perennial breeding, animal welfare management, high heterozygous genomes, complex polyploid genomes, expanded array of breeding methods, management event tracking, clonal propagation, repeated measures, and storing and using multiallelic genetic data.

DeltaBreed is architected from the BrAPI specification directly and thus is always compatible with the current BrAPI version.  When needed, Breeding Insight works with the BrAPI community to add additional endpoints to the specification.  DeltaBreed’s first successful integration with BrAPI was to Field Book, which allowed the pulling of trials and traits to Field Book and the push of collected data back to DeltaBreed.  Currently, testing is underway for DeltaBreed’s integration with DArT for BrAPI-mediated genotyping sample submission directly, and a BrAPI integration with Gigwa for multiallelic genetic data storage. DeltaBreed is publicly available now in a pre-release beta format with the full public release planned for summer 2025.

Figure x: High-level schematic of DeltaBreed's backend functionality.

![Figure x](https://github.com/plantbreeding/BrAPI-Manuscript2/assets/50887273/6b3dd0aa-7aa3-4644-8eb1-3cfaf44c278f)


#### BMS

<!-- Peter S: Stub paragraph to stimulate the writing process. Please edit, rewrite, or delete as needed. -->
[brapi sync](https://github.com/IntegratedBreedingPlatform/brapi-sync) and other things

#### Breedbase

<!-- Peter S: Stub paragraph to stimulate the writing process. Please edit, rewrite, or delete as needed. -->
Since the inception of BrAPI in 2014, the Breedbase group has been a heavy contributor to the BrAPI community.

#### BIMS

<!-- Peter S: Stub paragraph to stimulate the writing process. Please edit, rewrite, or delete as needed. -->
BIMS is relatively new to the BrAPI community, with their primary focus being a link to the Field Book mobile data collection app.

#### Germinate

<!-- Peter S: Stub paragraph to stimulate the writing process. Please edit, rewrite, or delete as needed. -->
Germinate is a combination breeding database and germplasm management database. 
